containers/app/app.py

```diff
@@ -0,0 +1,56 @@
+import os
+import subprocess
+import connexion
+import sqlalchemy
+from shapely.geometry import Polygon
+from shapely import wkt
+
+
+DB_PASSWORD = os.environ['POSTGRES_PASSWORD']
+RAW_DIR = '/osm/pbf'
+CONVERTED_DIR = '/osm/pbf/converted'
+MAPCONFIG_PATH = '/osm/mapconfig.xml'
+engine = sqlalchemy.create_engine(f'postgresql+psycopg2://postgres:{DB_PASSWORD}@pgrouting/postgres')
+
+
+class Loader:
+    @staticmethod
+    def create_dirs(directory):
+        if not os.path.exists(directory):
+            os.makedirs(directory)
+
+    @staticmethod
+    def convert(raw_directory, converted_directory):
+        for file in os.listdir(raw_directory):
+            if file.endswith(".osm.pbf"):
+                path = os.path.join(raw_directory, file)
+                osm_name = file.rstrip('.pbf')
+                new_path = os.path.join(converted_directory, osm_name)
+                subprocess.check_call([f'osmconvert {path} --drop-author --drop-version --out-osm -o={new_path}'], shell=True)
+
+    @staticmethod
+    def load(converted_directory, mapconfig_path, password):
+        for file in os.listdir(converted_directory):
+            path = os.path.join(converted_directory, file)
+            subprocess.check_call([f'osm2pgrouting --host pgrouting --f {path} --conf {mapconfig_path} --dbname postgres '
+                                   f'--username postgres --password {password}'], shell=True)
```

Comment:
And again - you don't realy understand what does this function load exactly
```suggestion
    @staticmethod
    def load_osm_files_to_db(osm_file_directory: str, mapconfig_path: str, password: str):
        for file in os.listdir(osm_file_directory):
            path = os.path.join(osm_file_directory, file)
            subprocess.check_call([f'osm2pgrouting --host pgrouting --f {path} --conf {mapconfig_path} --dbname postgres '
                                   f'--username postgres --password {password}'], shell=True)
```